Sources/MultipeerKit/Public API/MultipeerTransceiver.swift

```diff
@@ -25,8 +25,8 @@ public final class MultipeerTransceiver {
     public var peerDisconnected: (Peer) -> Void = { _ in }
     
     /// The current device's peer id
-    public var localPeerId: String? {
-        return connection.getLocalPeerId()
+    public var localPeer: Peer? {
```

Comment:
Let's re-add the `localPeerId` property and mark it as deprecated so that this doesn't break existing clients.

Reply:
Done - I guess ;)